server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -157,3 +176,45 @@ def grant_privilege(
             )
 
         return access_control_privilege_return
+
+
+class ForestClientValidator:
+    """
+    Purpose: More validations on inputs (other than basic validations) and
+             business rules validations (if any).
```

Comment:
Probably need to adjust this comment from previous one.
Previous validator was intended to validate additional business rules for the whole request object for the endpoint but here it is only for forest_client_number validation.

Reply:
updated